apps/src/code-studio/pd/downloadCsv.js

```diff
@@ -10,13 +9,21 @@ import _ from 'lodash';
  *        and the values will be used as column headings.
  */
 const downloadCsv = ({data, filename, headers}) => {
-  const includedKeys = Object.keys(headers);
   const exportData = data.map(row => {
-    row = _.pick(row, includedKeys);
-    row = _.mapKeys(row, (_, key) => headers[key]);
-    row = _.mapValues(row, value => value || ""); // replace null with empty string
+    const exportRow = {};
```

Comment:
should this be `let` instead of `const` because you call `exportRow[headers[key]] = value;` on line 23, or is that ok because you are modifying it but not reassigning it?

Reply:
Not reassigning, so const is appropriate. 

From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

> The const declaration creates a read-only reference to a value. It does not mean the value it holds is immutable, just that the variable identifier cannot be reassigned. For instance, in the case where the content is an object, this means the object's contents (e.g., its parameters) can be altered.